README.md

```diff
@@ -1,2 +1,102 @@
 # ePlant
-ePlant
+ePlant is a gene-first visualization tool for plant genomes.
```

Comment:
"gene-centered" instead of gene-first?

Reply:
i was using Vincent's language from the [onboarding blueprint](https://github.com/BioAnalyticResource/ePlant/issues/29) but am happy to change!